sandpack-react/src/components/Preview/index.tsx

```diff
@@ -32,6 +33,7 @@ export interface PreviewProps {
   showOpenInCodeSandbox?: boolean;
   showRefreshButton?: boolean;
   showSandpackErrorOverlay?: boolean;
+  additionalButtons?: JSX.Element;
```

Comment:
I'm not fully confident with the prop name. What about?
- Use `children` instead?
- `renderActionSection`;
- `renderActions`;
- `renderChildrenActions`;
- `actionsChildren`

What do you think?